app/mailers/prison_mailer.rb

```diff
@@ -107,4 +107,14 @@ def smoke_test?
   def visitors_email_address
     visit.visitors.first.email
   end
+
+  def first_date
+    Date.parse(@visit.slots.first.date)
+  end
+
+  def request_date
+    first_date.strftime('%A %e %B')
+  end
+
+  alias_method :receipt_date, :request_date
```

Comment:
Should these be the same? @tatyree, isn't this what you were talking about this morning?